services/filemanager/filemanager.go

```diff
@@ -51,31 +52,31 @@ func New(settings *SettingsT) (FileManager, error) {
 }
 
 // GetProviderConfigFromEnv returns the provider config
-func GetProviderConfigFromEnv() map[string]interface{} {
+func GetProviderConfigFromEnv(prefix string) map[string]interface{} {
```

Comment:
Thinking if we should use the same bucket as JOBS_BACKUP and use a default prefix. Fewer things to configure, but we will loose the flexibility. WDYT?

Reply:
Lot of users might not manadate storing of gateway dumps but expect us to store failed jobs/errors ?